Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Sensor/Radar.cpp

```diff
@@ -135,6 +141,7 @@ void ARadar::SendLineTraces(float DeltaTime)
     Rays[i].Hitted = false;
   }
 
+  auto LockedPhysObject = FPhysicsObjectExternalInterface::LockRead(GetWorld()->GetPhysicsScene());
```

Comment:
Did you profiled to verify that this optimizes the time execution time?

Adding a lock could increase the execution time even to add parallelization.

Reply:
We are not attempting to make a optimization yet. Just enabling radar as similar as possible to UE4 code, Jose has verified that LineTraces are slower in UE5 than UE4. We will need to look for a solution